nx.json

```diff
@@ -109,6 +109,11 @@
           "codeCoverage": true
         }
       }
+    },
+    "@nx/js:swc": {
```

Comment:
added defaults for swc executor ( which is used in some of our tooling packages )